### ISSUE TITLE
First Day of Week

### ISSUE THREAD

**Issue by flowhorn** [johnpryan/date_utils#2]
Hello,
I really like your work, especially the calendar widget for flutter, as i use it in my app. And it is wonderful. But I would like to have the calendar not starting sunday and ending saturday, but from monday to sunday.
This would be fantastic!

**Comment by Sadoge** [comment]
+1
It would be awesome if you could give us an option with which day of the week we would like to start. Thanks for the great work tho!

**Comment by JoeyMontero** [comment]
This is already a fantastic proyect, but having this option woudl be amazing.

**Comment by rravithejareddy** [comment]
Yes! For weekly calendar it should start from Monday, then only it will be useful. lot of other calendar providers doing that otherwise integrations will become tough.

**Comment by ghazi256** [comment]
Set firstWeekday = 1 in DateModel

and replace following code

`func indexAtBeginning(in month: MonthType) -> Int?{
        if let index = calendar.ordinality(of: .day, in: .weekOfMonth, for: atBeginning(of: month)) {
            let finalIndex = index - 2
            if finalIndex < 0{
                return 7 + finalIndex // +  because final index is in negative
            }
            return finalIndex
        }
        return nil
    }`
